### ISSUE TITLE
Site broken (embed.js fatal)

### ISSUE THREAD

**Issue by Krinkle** [travis-ci/travis-ci#6382]
For the past 5 minutes, all travis-ci.org urls yield the same blank grey page with an uncaught JS exception:

![screen shot 2016-07-29 at 21 17 12](https://cloud.githubusercontent.com/assets/156867/17262184/de1ef86e-55d1-11e6-9641-4fd9b8419217.png)

Browser: Chrome 54.0.2810.2 canary (Mac OS X)

It reports the error as `vendor-...` first, but it then changes to `ember.prod.js` (presumably from a source map):

> ```
> vendor-91d8890….js:11Uncaught TypeError: Cannot assign to read only property 'stack' of object '[object Object]'t @ vendor-91d8890….js:11A @ vendor-91d8890….js:13u @ vendor-91d8890….js:18trigger @ vendor-91d8890….js:18send @ vendor-91d8890….js:13sendToApp @ auth.js:232setData @ auth.js:132autoSignIn @ auth.js:61init @ auth-dependent.js:12r @ vendor-91d8890….js:12r @ vendor-91d8890….js:15create @ vendor-91d8890….js:15v @ vendor-91d8890….js:6l @ vendor-91d8890….js:6lookup @ vendor-91d8890….js:6lookup @ vendor-91d8890….js:14_setupLocation @ vendor-91d8890….js:13setupRouter @ vendor-91d8890….js:13startRouting @ vendor-91d8890….js:13startRouting @ vendor-91d8890….js:7didBecomeReady @ vendor-91d8890….js:7invoke @ vendor-91d8890….js:6flush @ vendor-91d8890….js:6flush @ vendor-91d8890….js:6end @ vendor-91d8890….js:5run @ vendor-91d8890….js:5join @ vendor-91d8890….js:5l.join @ vendor-91d8890….js:12(anonymous function) @ vendor-91d8890….js:12l @ vendor-91d8890….js:4fireWith @ vendor-91d8890….js:4ready @ vendor-91d8890….js:4a @ vendor-91d8890….js:3
```
>

> ```
> ember.prod.js:18920Uncaught TypeError: Cannot assign to read only property 'stack' of object '[object Object]'
t	@	ember.prod.js:18920
A	@	ember.prod.js:29772
u	@	ember.prod.js:49135
trigger	@	ember.prod.js:47873
send	@	ember.prod.js:29188
sendToApp	@	auth.js:232
setData	@	auth.js:132
autoSignIn	@	auth.js:61
init	@	auth-dependent.js:12
r	@	ember.prod.js:24542
r	@	ember.prod.js:37143
create	@	ember.prod.js:37414
v	@	ember.prod.js:1486
l	@	ember.prod.js:1334
lookup	@	ember.prod.js:1254
lookup	@	ember.prod.js:33620
_setupLocation	@	ember.prod.js:29278
setupRouter	@	ember.prod.js:28971
startRouting	@	ember.prod.js:28956
startRouting	@	ember.prod.js:3888
didBecomeReady	@	ember.prod.js:4856
invoke	@	ember.prod.js:990
flush	@	ember.prod.js:1054
flush	@	ember.prod.js:862
end	@	ember.prod.js:176
run	@	ember.prod.js:298
join	@	ember.prod.js:318
l.join	@	ember.prod.js:23512
(anonymous function)	@	ember.prod.js:23575
l	@	jquery.js:3187
fireWith	@	jquery.js:3317
ready	@	jquery.js:3536
a	@	jquery.js:3552
> ```

**Comment by BanzaiMan** [comment]
Does the problem occur on Chrome, or any other browser?

**Comment by Krinkle** [comment]
It happens consistently in Chrome 54 canary (Mac OS X). However it does not seem to happen in Chrome 51 stable.

I'd recommend keeping an eye on this. Maybe filing it upstream to Embed or Chrome, depending on what is causing it. If things go on as-is, it will result in global failure for all Chrome users in a few weeks.

**Comment by Kwpolska** [comment]
Experiencing the same issue in Version 54.0.2810.2 dev (64-bit) on Linux.

**Comment by jstockwin** [comment]
And likewise on Version 54.0.2810.2 dev-m, but on Windows this time. It's fine on Firefox.

**Comment by backspace** [comment]
Hey, this is fixed on Chrome 54.0.2815.0. See [this Ember thread](https://github.com/emberjs/ember.js/issues/13919) for more information.
